spaceship/views.py

```diff
@@ -95,6 +98,15 @@ def login():
 def about():
   return render_template('about.html')
 
+@app.route('/mission/<mission_id>', methods=['GET', 'POST'])
+def show_mission_info(mission_id):
+  # use the mission's title (lowercase, replacing spaces with underscores)
+  # as the filename of the mission info template
+  mission_title = get_mission_title(mission_id)
```

Comment:
This is cool but we could also add a db field to track the more info url at some point.